tests/test_user.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for users"""
+from tests.base import BaseTestCase
+from app import app 
+from passlib.hash import pbkdf2_sha256
+
+import unittest
+import psycopg2
+import json
+
+class TestRequestTestCase(BaseTestCase):
+    """ Test for normal user"""
+    def setUp(self):
```

Comment:
The setup is not needed here since you have it in the BaseTestCase class.

Reply:
okay i removed it